.github/workflows/test-e2e.yml

```
@@ -6,11 +6,20 @@ on:
       - master
 
 jobs:
+  chrome_e2e_previous_major:
+    name: Chrome with previous major WP version.
+    uses: ./.github/workflows/test-e2e-cypress.yml
+    with:
+      wpVersion: "WordPress/WordPress#6.2.2"
```

Previous major release (in the world of WP) would be 6.2
```suggestion
      wpVersion: "WordPress/WordPress#6.2"
```

---

Good point!